examples/experimental/trial/gbt_estimator/model_def.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import tensorflow as tf
+
+from typing import Callable, Dict, List, Tuple
+
+from determined.estimator import EstimatorTrial, EstimatorTrialContext
+
+class BoostedTreesTrial(EstimatorTrial):
+    def __init__(self, context: EstimatorTrialContext) -> None:
+        self.context = context
+
+        # Load Dataset
```

Comment:
non-blocking: add periods at the end of comments.

Reply:
Done.